tests/test_test_utils.py

```diff
@@ -58,7 +69,39 @@ def test_get_route():
                 resp = yield from client.request("GET", "/")
                 assert resp.status == 200
                 text = yield from resp.text()
-                assert "Hello, world" in text
+                assert _hello_world_str == text
+
+            loop.run_until_complete(test_get_route())
+
+
+def test_auto_gzip_decompress():
+    with loop_context() as loop:
+        app = _create_example_app()
+        with _TestClient(app, loop=loop) as client:
```

Comment:
@asvetlov  Here is the problem. If you really want to use TestClient directly, you need a  TestServer instance. It's best to use the `test_client` fixture if you can:

```python
def test_auto_gzip_decompress(test_client):
    app = _create_example_app() 
    with test_client(app) as client:
```

Otherwise:

```python
     with loop_context() as loop:
         app = _create_example_app()
         with _TestClient(_TestServer(app, loop=loop), loop=loop) as client:
```